rafthttp/pipeline.go

```diff
@@ -150,6 +150,7 @@ func (p *pipeline) post(data []byte) (err error) {
 	}()
 	done := make(chan struct{}, 1)
 	go func() {
+		waitSchedule()
```

Comment:
good idea. moving to line 157 let it skip waiting when the request is done in less than 1ms. will change.